script/DeployZkMinterDelayV1.ts

```diff
@@ -0,0 +1,44 @@
+import { config as dotEnvConfig } from "dotenv";
+import { Deployer } from "@matterlabs/hardhat-zksync-deploy";
+import { Wallet } from "zksync-ethers";
+import * as hre from "hardhat";
+
+const MINTABLE_ADDRESS = ""; // TODO: Update this to the actual mintable address.
+const ADMIN_ACCOUNT = ""; // TODO: Update this to the actual admin account.
+const MINT_DELAY = 86400; // TODO: Update this to the actual mint delay. Currently set to 24 hours.
+const SALT = ""; // TODO: Update this to the actual salt.
+
+async function main() {
+  dotEnvConfig();
+
+  const deployerPrivateKey = process.env.DEPLOYER_PRIVATE_KEY;
+  if (!deployerPrivateKey) {
+    throw "Please set DEPLOYER_PRIVATE_KEY in your .env file";
+  }
+
+  const contractName = "ZkMinterDelayV1";
+  console.log("Deploying " + contractName + "...");
```

Comment:
These look good! Did you test them on sepolia?

Reply:
ZkMinterDelayV1Factory was deployed to `0xf6Fc6B46A29A20a5A0bbD280294B44B259c673C2`
ZkMinterDelayV1 was deployed to `0x469C7d2dA81213D3757429213B8935aE14b0F23D`